data/Debian.yaml

```diff
@@ -7,5 +7,5 @@ rundeck::repo_config:
     release: any
     repos: main
     key:
-      name: rundeck
-      source: https://packages.rundeck.com/pagerduty/rundeck/gpgkey
+      id: '0DDD2FA79B15D736ECEA32B89B5206167C5C34C0'
+      server: 'keyserver.ubuntu.com'
```

Comment:
This generates obsolete sources.list entries, use `name: rundeck.asc` instead.

Reply:
yh not sure what's going wrong but it's not working with the new apt method i will try again but it will keep saying it doesn't find the proper key

Reply:
I would have to rewrite the code or do a hiera lookup or something which al seems a bit much if the public gpg key is actually available